### PR TITLE
Remove gem Slim

### PR DESCRIPTION
**Pull request by dinesh-skyach**
Recoded Slim templates to ERB and removed gem slim #823

### CODE REVIEW

File: app/views/kaminari/_prev_page.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Link to the "Previous" page
+- available local variables
+  url          : url to the previous page
+  current_page : a page object for the currently displayed page
+  total_pages  : total number of pages
+  per_page     : number of items to fetch per page
+  remote       : data-remote
+-->
+
+<li class="previous <%= (current_page.first?)? 'disabled' : '' %>">
```
Comment (yivo) [inline app/views/kaminari/_prev_page.html.erb]: Please, use `<%= 'disable' if current_page.first? %>` everywhere. It is hard to understand this line.

File: app/views/kaminari/_prev_page.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<%= link_to raw(t 'views.pagination.previous'), url, :rel => 'prev', :remote => remote %>
```
Comment (yivo) [inline app/views/kaminari/_prev_page.html.erb]: Use new Ruby hash syntax. Pls update everywhere.

File: app/views/kaminari/_paginator.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  <ul class="pagination">
+    <%= first_page_tag %>
+    <%= prev_page_tag %>
+    <% each_page do |page| %>
+      <% if page.left_outer? || page.right_outer? || page.inside_window? %>
+        <%= page_tag page %>
+      <% elsif !page.was_truncated? %>
+        <%= gap_tag %>
+      <% end %>
+      <% end %><%= next_page_tag %>
```
Comment (yivo) [inline app/views/kaminari/_paginator.html.erb]: @dinesh-skyach Please maintain positions of code: `<%= next_page_tag %>` should be on the next line.

File: app/views/kaminari/_paginator.html.erb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<!--
+The container tag
+- available local variables
```
Comment (yivo) [inline app/views/kaminari/_paginator.html.erb]: @dinesh-skyach I see the problems with the alignment of comments. Please, check original comments from slim.

**Review by yivo — CHANGES_REQUESTED** [review]
@dinesh-skyach I see many issues with code style in pagination views. Have you used automatic tools for converting slim to erb? I would not use them since they produce bad results. I think we need to try generate ERB views by using kaminari gem. Please update the gem to the latest version, generate erb views using generator from the gem and test paginators in admin panel.

**Comment by dinesh-skyach** [comment]
@yivo, I have made the changes.

**Review by yivo — APPROVED** [review]

**Pull request merged**